### PR TITLE
Crash details page: crash map

### PR DESCRIPTION
**Pull request by roseeichelmann**
## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/17182

## Testing
**URL to test:** 
Local

**Steps to test:**
1. Follow instructions in https://github.com/cityofaustin/atd-vz-data/pull/1440 to start the data model collector branch and run the CRIS import ETL, or if you already have it set up use that!
2. Go to a crash, see that the crash map says "Geocode Provider: TxDOT CRIS"
3. Edit the crash coordinates, move it to some other road. See the location id change and it will now say "Geocode Provider: Manual Q/A".

---
#### Ship list
- [ ] Check migrations for any conflicts with latest migrations in master branch
- [ ] Confirm Hasura role permissions for necessary access
- [ ] Code reviewed
- [ ] Product manager approved

### CODE REVIEW

File: atd-vze/src/views/Crashes/Crash.js [hunk 1]
```diff
@@ -73,24 +73,6 @@ function Crash(props) {
...
-    let geocoderAddressString = "";
-    geocoderAddressFields.forEach(field => {
-      if (data?.atd_txdot_crashes?.[0]?.[field] !== null) {
-        geocoderAddressString = geocoderAddressString.concat(
-          data?.atd_txdot_crashes?.[0]?.[field] + " "
-        );
-      }
-    });
-    return geocoderAddressString;
-  };
```
Comment (roseeichelmann) [inline atd-vze/src/views/Crashes/Crash.js:92]: this was not actually being used for anything?

**Review by johnclary — CHANGES_REQUESTED** [review]
this is fantastic! i flagged a few minor things.

**Comment by roseeichelmann** [comment]
@johnclary ready for re-review! worth checking to make sure none of your change log stuff that got merged into `data-model-v2` was affected but im pretty sure there were no regressions thru my own testing and looking at the code diff in this PR

File: atd-vze/package-lock.json
Comment (johnclary) [inline atd-vze/package-lock.json:1]: i think this is fine 🤷

**Review by johnclary — APPROVED** [review]
yep—this is ready to merge! thanks @roseeichelmann! 🚢 🚢 🚢 🚢 🚢

File: atd-vze/package-lock.json
Comment (roseeichelmann) [reply atd-vze/package-lock.json:1]: me too :shrug:

**Pull request merged**